app/api/chat/route.ts

```
@@ -281,12 +263,107 @@ export async function POST(request: Request) {
+            }
+          }
+
+          // Format citations if we found any
+          if (citations.length > 0) {
+            // Add a separator and sources section
+            finalText += "\n\n---\n\n**Sources:**\n";
+
+            // Deduplicate sources by URL
+            const uniqueSources = new Map<string, Citation>();
+            citations.forEach((citation: Citation) => {
+              if (citation.source && citation.source.uri) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)